tests/views_integration/Tribe/Events/Views/V2/Partials/Day_View/Event/DateTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Tribe\Events\Views\V2\Partials\Day_View\Event;
+
+use Tribe\Test\PHPUnit\Traits\With_Post_Remapping;
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlPartialTestCase;
+
+class DateTest extends HtmlPartialTestCase
+{
+	use With_Post_Remapping;
+
+	protected $partial_path = 'day/event/date';
+
+	/**
+	 * Test render with event
+	 */
+	public function test_render_with_event() {
+		$event = $this->mock_event( 'events/single/1.json' )->get();
```

Comment:
You can just use `get_mock_event` directly here:
```suggestion
		$event = $this->get_mock_event( 'events/single/1.json' );
```